src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx

```diff
@@ -334,6 +344,37 @@ const ActionsPageEvent = ({
                   </span>
                 ),
                 storageSlot: values?.slot?.toHexString(),
+                amountTag: (
+                  <Tag
+                    appearance={{
+                      theme: 'primary',
+                      colorSchema: 'inverted',
+                      fontSize: 'tiny',
+                      margin: 'none',
+                    }}
+                  >
```

Comment:
Please apply the following css to the inline tags for this message:
```css
    vertical-align: top;
    margin-top: 2px;
```
This will align the tags properly with the text.

Here's how it looks currently:
![Screenshot from 2021-04-20 12-10-54](https://user-images.githubusercontent.com/1193222/115370191-7d752c80-a1d1-11eb-9a73-f8f84848ce38.png)

And here's how it looks with the tags aligned
![Screenshot from 2021-04-20 12-04-12](https://user-images.githubusercontent.com/1193222/115370224-8665fe00-a1d1-11eb-8846-e8b87d3a5043.png)

If you decide to wrap it in another element, make sure to also apply `display: inline-block;` to that element as well